plugin/kubernetes/xfr_test.go

```diff
@@ -15,7 +15,7 @@ import (
 func TestKubernetesXFR(t *testing.T) {
 	k := New([]string{"cluster.local."})
 	k.APIConn = &APIConnServeTest{}
-	k.TransferTo = []string{"127.0.0.1"}
```

Comment:
if you don't specify the port in the setup, it still works?

Reply:
I recall that it runs it through a normalizer, which adds the default 53 port...

I'll double check  ...

Reply:
https://github.com/coredns/coredns/blob/master/plugin/pkg/parse/host.go#L82 Yeah it should be fine to specify w/o a port.